Foundation/CGFloat.swift

```diff
@@ -143,6 +143,10 @@ public struct CGFloat {
     
     /// The native value.
     public var native: NativeType
+
+    public var double: Double {
```

Comment:
This is an addition to API surface area, can this not be done in the call sites?